.ado/templates/react-native-init.yml

```diff
@@ -70,32 +70,40 @@ steps:
     inputs:
       versionSpec: ">=4.6.0"
 
-  # We force the usage of npm instead of yarn becuase yarn has fragility issues when redirected to a different server (such as verdaccio)
-  - task: CmdLine@2
-    displayName: Init new project
-    inputs:
-      script: npx react-native init testcli --npm --template react-native@$(reactNativeDevDependency)
-      workingDirectory: $(Agent.BuildDirectory)
-
   - task: DownloadBuildArtifacts@0
     displayName: 'Download Nuget Test Feed'
     inputs:
       artifactName: NuGetTestFeed
       downloadPath: $(System.DefaultWorkingDirectory)
     condition: ${{ parameters.useNuGet }}
+
+  # Alias yarn to midgard-yarn which has some fixes for yarns fragility when redirected to a different server (such as verdaccio)
+  - script: yarn global add midgard-yarn@1.23.14
+    displayName: Install midgard-yarn
+
+  - script: mkdir midgard
+    displayName: Make midgard alias dir
+    workingDirectory: $(Agent.BuildDirectory)
+
+  - script: echo midgard-yarn %* > yarn.cmd
+    displayName: Alias yarn -> midgard-yarn in root
+    workingDirectory: $(Agent.BuildDirectory)/midgard
+
+  - script: setx PATH "$(Agent.BuildDirectory)/midgard;%PATH%"
+    displayName: Add midgard alias dir to path dirs
     
-  - task: CmdLine@2
-    displayName: Apply windows template (with nuget)
-    inputs:
-      script: npx react-native-windows-init --verbose --version $(npmVersion) --overwrite --language ${{ parameters.language }} ${{ parameters.additionalInitArguments }} --experimentalNuGetDependency true --nuGetTestFeed $(System.DefaultWorkingDirectory)\NuGetTestFeed
-      workingDirectory: $(Agent.BuildDirectory)\testcli
+  - script: npx react-native init testcli --template react-native@$(reactNativeDevDependency)
+    displayName: react-native init
+    workingDirectory: $(Agent.BuildDirectory)
+    
+  - script: npx react-native-windows-init --verbose --version $(npmVersion) --overwrite --language ${{ parameters.language }} ${{ parameters.additionalInitArguments }} --experimentalNuGetDependency true --nuGetTestFeed $(System.DefaultWorkingDirectory)\NuGetTestFeed
+    displayName: react-native-windows-init
+    workingDirectory: $(Agent.BuildDirectory)/testcli
     condition: ${{ parameters.useNuGet }}
```

Comment:
If you're here, would you mind adding `succeeded` to the condition of this and below so we don't try to run this after previous steps have failed?